platform-includes/distributed-tracing/custom-instrumentation/javascript.mdx

```diff
@@ -205,6 +205,46 @@ In this example, tracing information is propagated to the project running at `ht
 
 The two services are now connected with your custom distributed tracing implementation.
 
+<PlatformCategorySection supported={['server', 'serverless']}>
+
+### Injecting Tracing Information into HTML
+
+If you're server-side rendering HTML and you use a Sentry SDK in your browser application, you can connect the backend and frontend traces.
+You can do this by injecting your server's tracing information `<meta>` tags into the HTML that's initially served to the client.
+When the frontend SDK is initialized, it will automatically pick up the tracing information from the `<meta>` tags and continue the trace.
+Note that your browser SDK needs to register `browserTracingIntegration` for this to work.
```

Comment:
```suggestion
If you're server-side rendering HTML and you use a Sentry SDK in your browser application, you can connect the backend and frontend traces by injecting your server's tracing information `<meta>` tags into the HTML that's initially served to the client. When the frontend SDK is initialized, it will automatically pick up the tracing information from the `<meta>` tags and continue the trace. Note, that your browser SDK needs to register `browserTracingIntegration` for this to work.
```